include/RAJA/util/Registry.hpp

```diff
@@ -86,7 +86,7 @@ namespace util {
 
     // begin is not defined here in order to avoid usage of an undefined static
     // data member, instead it's instantiated by RAJA_INSTANTIATE_REGISTRY.
-    static iterator begin();
+    static RAJASHAREDDLL_API iterator begin();
```

Comment:
Are these the only places that we will have to add this macro? Or, will new locations pop up based on usage?

Reply:
I'm not entirely sure.  It should not be needed for anything that is inlined.